tests/shipping/models/Pickup Invoice Address - Credit card.model.js

```diff
@@ -0,0 +1,100 @@
+import { setup, visitAndClearCookies } from '../../../utils'
+import {
+  fillEmail,
+  getRandomEmail,
+  fillProfile,
+} from '../../../utils/profile-actions'
+import { fillPickupAddress, goToPayment } from '../../../utils/shipping-actions'
+import {
+  completePurchase,
+  payWithCreditCard,
+} from '../../../utils/payment-actions'
+import {
+  goToInvoiceAddress,
+  fillInvoiceAddress,
+} from '../../../utils/invoice-actions'
+import { ACCOUNT_NAMES, SKUS } from '../../../utils/constants'
+
+export default function test(account) {
+  describe(`Pickup Invoice Address - Credit card - ${account}`, () => {
+    before(() => {
```

Comment:
Same as https://github.com/vtex/checkout-ui-tests/pull/63#discussion_r463640383.
> I think this should be a beforeEach to keep consistency between tests.
>
> The reason why all the other tests use before is because they have only a single it per test plan, so it might be a little bit misleading.

```suggestion
    beforeEach(() => {
```

Reply:
oh, nice, thanks